disco_aws_automation/resource_helper.py

```diff
@@ -211,3 +201,18 @@ def check_written_s3(object_name, expected_written_length, written_length):
         raise S3WritingError(
             "{0} is not written correctly to S3 bucket".format(object_name)
         )
+
+
+def backoff(cycle):
+    """
+    This function takes as input an integer that represents a cycle count,
+    calculates jitter and executes sleep for the calculated time.
+    The value of 'cycle' must be an integer greater than 0.
+    """
+    base = 3
+    max_poll_interval = 60
+    if cycle is None or cycle < 1:
+        raise ValueError('Value of cycle must be int > 0')
+
+    time.sleep(min(max_poll_interval, randint(base, cycle * 3)))
+    return cycle + 1
```

Comment:
This still needs to return the time so that the time passed variable makes sense.